### PR TITLE
PEP 686: Update based on discussion

### PR DESCRIPTION
**Pull request by methane**

### CODE REVIEW

File: pep-0686.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ UTF-8 becomes de-facto standard text encoding.
   default.
 * Most websites and text data on the internet uses UTF-8.
 * And many other popular programming languages including node.js, Go, Rust,
-  Ruby, and Java uses UTF-8 by default.
+  and Java uses UTF-8 by default.
```
Comment (methane) [inline pep-0686.rst:33]: I removed Ruby here because Ruby still uses locale encoding on Unix.

**Review by CAM-Gerlach — COMMENTED** [review]
LGTM from a purely technical/PEP 12 perspective, thanks. If you'd like me to help copyedit/proofread the text for grammar, etc, I'm happy to do so (it would probably be best to do so as a separate PR for your review once this is merged); just let me know.

**Review by ncoghlan — APPROVED** [review]

**Comment by ncoghlan** [comment]
This update cleanly addresses the concern I raised on Discourse.

**Comment by methane** [comment]
Thank you for review. I merge this for now to continue more discussions.

**Pull request merged**